bower.json

```diff
@@ -16,6 +16,7 @@
     "purescript-console": "^3.0.0",
     "purescript-math": "^2.0.0",
     "purescript-vector": "^2.0.0",
+    "purescript-matrix": "^2.0.0",  
```

Comment:
I considered this an innocuous addition as it's a sibling to Data.Vector, which is already used. I can't tell that purescript-matrix actually depends on mjs, perhaps the docs are out of date.